02week/tests.js

```diff
@@ -0,0 +1,45 @@
+if (typeof describe === 'function') {
+
+	describe('#rockPaperScissors()', () => {
+		it('should detect a tie', () => {
+			assert.equal(rockPaperScissors('rock', 'rock'), "It's a tie!");
+			assert.equal(rockPaperScissors('paper', 'paper'), "It's a tie!");
+			assert.equal(rockPaperScissors('scissors', 'scissors'), "It's a tie!");
+		});
+		it('should detect that hand one won', () => {
+			assert.equal(rockPaperScissors('rock', 'scissors'), "Hand one wins!");
+			assert.equal(rockPaperScissors('paper', 'rock'), "Hand one wins!");
+			assert.equal(rockPaperScissors('scissors', 'paper'), "Hand one wins!");
+		});
+		it('should detect that hand two won', () => {
+			assert.equal(rockPaperScissors('rock', 'paper'), "Hand two wins!");
+			assert.equal(rockPaperScissors('paper', 'scissors'), "Hand two wins!");
+			assert.equal(rockPaperScissors('scissors', 'rock'), "Hand two wins!");
+		});
+		it('should detect that hand two won', () => {
+			assert.equal(rockPaperScissors('rock', 'paper'), "Hand two wins!");
+			assert.equal(rockPaperScissors('paper', 'scissors'), "Hand two wins!");
+			assert.equal(rockPaperScissors('scissors', 'rock'), "Hand two wins!");
+		});
+		it('should detect a valid entry from user', () => {} (
```

Comment:
This text is inputting the same arguments as the tests above, but expecting a different output.